### PR TITLE
Fix and improve README

### PR DESCRIPTION
**Pull request by szepeviktor**
preview https://github.com/szepeviktor/CodingStandard/blob/patch-1/README.md 👀

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,12 @@
...
 
-Installation & usage
---------------------
+## Installation & usage
 
 1. Install this package:
 
     ```bash
-    $ composer require --dev sylius-labs/coding-standard
+    composer require --dev sylius-labs/coding-standard
```
Comment (szepeviktor) [inline README.md:12]: Make this copyable.

File: README.md [hunk 2]
```diff
@@ -19,19 +18,17 @@ Installation & usage
...
-        $ecsConfig->import('vendor/sylius-labs/coding-standard/ecs.php');
-    };
-    ```
+```php
+use Symplify\EasyCodingStandard\Config\ECSConfig;
+
+return static function (ECSConfig $ecsConfig): void {
+    $ecsConfig->import(__DIR__ . '/vendor/sylius-labs/coding-standard/ecs.php');
+};
+```
```
Comment (szepeviktor) [inline README.md:29]: Fix Markdown

Comment (szepeviktor) [reply README.md:29]: and use absolute paths

**Review by coldic3 — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Viktor! :1st_place_medal:

**Comment by szepeviktor** [comment]
You're welcome.

**Comment by szepeviktor** [comment]
It was fun to explore your bytes.

**Pull request merged**